### PR TITLE
Release/v0.3.0

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

## 🚨 Breaking Change and 🎉 Feature
- Intermediate models now materialize in their own schema named `recurly_intermediate`. This is to reduce noise in the schema for the final model outputs. ([#14](https://github.com/fivetran/dbt_recurly/pull/14))
## 🔧 Bug Fixes
- Updated how `date_week` is determined in model `int_recurly__account_rolling_totals`. ([#14](https://github.com/fivetran/dbt_recurly/pull/14))

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes (It was already listed as 0.2.1 in the yml,  but I didn't see a corresponding release, so I left it.)

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature #13  #12 
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🪴

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-avinash — CHANGES_REQUESTED** [review]
@fivetran-catfritz All looks good! Just a few minor suggestions & changes and you are good to go.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# dbt_recurly v0.3.0
+## 🚨 Breaking Change and 🎉 Feature
+- Intermediate models now materialize in their own schema named `recurly_intermediate`. This is to reduce noise in the schema for the final model outputs. ([#15](https://github.com/fivetran/dbt_recurly/pull/15))
```
[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply CHANGELOG.md:3]: I ended up only creating the release branch, so unfortunately I don't have another PR to link to. Not sure if I should link the issue you mentioned instead?

File: dbt_project.yml [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-version: '0.2.1'
+version: '0.3.0'
 require-dbt-version: [">=1.3.0", "<2.0.0"]
 
 models:
   recurly:
     +schema: recurly
     +materialized: table
     intermediate:
+      +schema: recurly_intermediate
```
[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply dbt_project.yml:12]: done.

[the remark this answers is not in the document]
Comment (fivetran-catfritz) [reply dbt_project.yml]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+  - **Note:** Before running this new version, we recommend that you manually remove any `int_recurly...` tables from the original output schema. 
+## 🔧 Bug Fixes
+- Updated how `date_week` is determined in model `int_recurly__account_rolling_totals`. ([#15](https://github.com/fivetran/dbt_recurly/pull/15))
```
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md:6]: My mistake! For some reason I thought that issue was the PR.

Disregard then. 

Make sure to assign yourself to that issue btw!

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fivetran-avinash) [reply CHANGELOG.md]: Oh interesting! For some reason I thought that issue was the PR.

Disregard then. 

Make sure to assign yourself to that issue btw!

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-catfritz Approved!

**Pull request merged**